sw/otbn/crypto/p384_a2b.s

```diff
@@ -133,13 +133,6 @@ p384_arithmetic_to_boolean_mod:
  * clobbered flag groups: FG0
  */
 p384_arithmetic_to_boolean:
-  /* Initialize inputs: in case of randomness in upper part of inputs
-     truncate to 385 bits. */
-  bn.rshi   w19, w19, w31 >> 129
-  bn.rshi   w19, w31, w19 >> 127
-  bn.rshi   w12, w12, w31 >> 129
-  bn.rshi   w12, w31, w12 >> 127
-
```

Comment:
Is there a reason why these bits will be ignored later on, or is it now a precondition of the function that they be 0?

Reply:
I was searching for possibilities to save imem, and I discovered that in the way the calling functions are written there won't be any randomness in the upper part of the input. Because the calling function only receives actual 384-bit values, there won't be randomness in the more significant bits. Therefore, I removed those lines. (Probably something that can be done for p256 too.)